ukvm/ukvm-cpu.h

```diff
@@ -42,19 +42,26 @@
  */
 #define X86_EFLAGS_CF	0x00000001 /* Carry Flag */
 
-/*
- * Basic CPU control in CR0
- */
-#define X86_CR0_PE_BIT		0 /* Protection Enable */
-#define X86_CR0_PE		_BITUL(X86_CR0_PE_BIT)
-#define X86_CR0_PG_BIT		31 /* Paging */
-#define X86_CR0_PG		_BITUL(X86_CR0_PG_BIT)
+
 
 /*
- * Intel CPU features in CR4
+ * CPU control registers
  */
-#define X86_CR4_PAE_BIT		5 /* enable physical address extensions */
-#define X86_CR4_PAE		_BITUL(X86_CR4_PAE_BIT)
+#define X86_CR0_PE 0x00000001 /* Protected mode Enable */
+#define X86_CR0_NE 0x00000020 /* Numeric Error enable (EX16 vs IRQ13) */
+#define X86_CR0_PG 0x80000000 /* PaGing enable */
```

Comment:
This might be a bit of a pain to change (and it's up to you), but using `_BITUL` made it easier to check for the flag number directly. For example, mapping 0x80000000 to 31 is not as "straightforward" as before.